src/compliment/sources/class_members.clj

```diff
@@ -80,9 +86,24 @@
   (get-in @members-cache [ns :members]))
 
 (defn class-member-symbol?
-  "Tests if a symbol name looks like a non-static class member."
-  [^String x]
-  (.startsWith x "."))
+  "Tests if `x` looks like a non-static class member,
+  e.g. \".getMonth\" or (with `clj-1-12+` option set) \"java.util.Date/.getMonth\".
+
+  When true, yields `[klass-name method-name]`."
+  ([x] (class-member-symbol? x nil))
+  ([^String x {:keys [clj-1-12+]}]
+   (when-let [[_ klass-name member-name]
+              (re-matches #"(?x)
+                            (?=.)  # at least one character
+                            (?:    # optional 'a.b/'
+                              ([^\/\:\.][^\:]*) # capture 'a.b'
+                              (?<!\.)  # deny 'a.b./'
+                              \/)?
+                            (?:    # optional '.method'
+                              \.([^.]*))?  # capture 'method'"
```

Comment:
You should just reuse `static-member-symbol?` here (either literally or inline it here) instead of introducing a new complicated regexp.